app/models/goal.py

```diff
@@ -1,6 +1,15 @@
 from flask import current_app
 from app import db
+from sqlalchemy.orm import backref
```

Comment:
I think we don't need this import statement. The `backref` on line 9 is just a named parameter for the relationship function instead of an object we need to import.